packages/protocol/contracts/libs/LibAddress.sol

```diff
@@ -18,10 +18,8 @@ library LibAddress {
      * @param amount The amount of Ether to send.
      */
     function sendEther(address to, uint256 amount) internal {
-        if (amount > 0) {
-            (bool success, ) = payable(to).call{value: amount}("");
-            require(success, "ETH transfer failed");
-        }
+        (bool success, ) = payable(to).call{value: amount}("");
+        require(success, "ETH transfer failed");
```

Comment:
391 (with >0 check) vs 3154 (without the check)

Reply:
I think this is for when the `call` taken vs not taken because the different is pretty big? I meant that now each time `sendEther` is called (quite a lot of places), the caller has to do the `amount > 0` check, instead of just using doing it here once, so that the caller doesn't have to bother checking it. The gas cost difference I think at worst could just be an extra internal call, at best the internal function is inlined so should be the same gas cost.

Reply:
https://github.com/taikoxyz/taiko-mono/pull/13390 Please see this PR.